app/gui/src/presenter.rs

```diff
@@ -29,6 +31,16 @@ pub use searcher::Searcher;
 
 
 
+// =================
+// === Constants ===
+// =================
+
+/// We don't know how long the project opening will take, but we still want to show a fake progress
+/// indicator for the user. This constant represents a progress percentage that will be displayed.
```

Comment:
What do you mean by "fake progress indicator"? What is the difference between a "fake" and a "real" progress indicator?

Reply:
> What do you mean by "fake progress indicator"? What is the difference between a "fake" and a "real" progress indicator?

To me, it's quite obvious, especially in the context: the "fake progress" is a progress which is not real progress, i.e. 80% does not mean that 80% of the stuff is done.

Reply:
Oh, this is about a "fake progress amount", not a "fake progress indicator"? The indicator is the spinner, I was wondering if we are talking here about a "fake spinner". So we are talking here about the real (not fake spinner) with a fake value, right?

Reply:
Yes, I just read it as `((fake progress) indicator)` while you seemingly `(fake (progress indicator))` ;) We may make a clarification there.

Reply:
This was just a comment that I copied over from the existing code that showed the spinner when switching projects. I can change it if it helps you.

Reply:
Well, it will be removed when I change it to progress over a hard-coded amount of time.